### PR TITLE
CI deploys the Admin Portal

### PR DESCRIPTION
**Pull request by emileswarts**
# What

This will automatically deploy the admin portal through CI.
Remove duplicated deployment logic and document in README.

# Why

We want changes pushed through the pipeline to be immediately deployed without having to manually restart the tasks in ECS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,53 @@ COGNITO_USER_POOL_SITE
...
+2. Deploy new tasks into the service
+
+### Deployment
+
+The `deploy` command is wrapped in a Makefile, it calls `./scripts/deploy` which schedules a zero downtime phased [deployment](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/update-service.html) in ECS.
+
+It doubles the currently running tasks and briefly serves traffic from the new and existing tasks in the service.
+The older tasks are eventually decomissioned, and production traffic is gradually shifted over to only the new running tasks.
+
+On CI this command is executed from the buildspec.yml file after migrations and publishing the new image to ECR has been completed.
```
Comment (neilkidd) [inline README.md:74]: We could add a link to buildspec.yml?

**Review by neilkidd — APPROVED** [review]
LGTM - subject to 1 tiny comment

File: README.md (hunk 1 above)
Comment (emileswarts) [reply README.md:74]: 👍

**Review by Themitchell — APPROVED** [review]
Nice. Looks good!

**Pull request merged**